platform/measurement_utils.cpp

```diff
@@ -245,6 +245,40 @@ string FormatSpeedUnits(Units units)
   UNREACHABLE();
 }
 
+// Interleaves the bits of two 32-bit numbers.
+// The result is known as a Morton code.
+long interleave_bits(long x, long y)
+{
+  long c = 0;
+  for(int i=31; i>=0; i--)
+  {
+    c = (c << 1) | ((x >> i) & 1);
+    c = (c << 1) | ((y >> i) & 1);
+  }
+
+  return c;
+}
```

Comment:
I suspect that we already have it: BitwiseMerge in bits.hpp
Can write unit tests if you doubt.